priv/www/js/help.js

```diff
@@ -328,6 +328,18 @@ HELP = {
         <dd>Rate at which queue index segment files are written. </dd>\
       </dl>',
 
+    'gc-operations':
+    'Number of garbage collection operations since the system started.',
```

Comment:
Is it since the system was started, though? We report it for the last N seconds.